src/common/transformations/tests/transpose_sinking/ts_general_test.cpp

```diff
@@ -380,7 +380,7 @@ TEST_F(TransformationTestsF, TSGeneralTestMultipleTypes) {
         auto ng_order0 = std::make_shared<Constant>(ov::element::u64, ov::Shape{4}, ov::Shape{0, 2, 3, 1});
         auto transpose0 = std::make_shared<Transpose>(node0, ng_order0);
 
-        auto reshape_const = std::make_shared<Constant>(ov::element::u64, ov::Shape{4}, ov::Shape{1, 40, 55, 96});
+        auto reshape_const = std::make_shared<Constant>(ov::element::u64, ov::Shape{4}, ov::Shape{2, 20, 55, 96});
```

Comment:
Wouldn't it be better to add a new test case instead of replacing the old one?

Reply:
It's not replacing the test. I used reshape in this test to prevent of moving Transpose through it. But now TransposeSinking moves through such Transposes. I just fixed reshape const this test to continue working as I expected